### PR TITLE
Add section-6 (Form section)

### PR DESCRIPTION
**Pull request by katsu223**
[Deploy link](https://katsu223.surge.sh)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -221,6 +221,53 @@ <h2 class="title review-title">Guest Reviews</h2>
...
+    <section class="form-section">
+      <div class="form-container">
+        <img class="form-image" src="resources/images/content/form-image-1.png">
+        <img src="resources/images/content/form-image-2.png">
+        <div class="booking-form">
+          <h2 class="form-title">Just the right food</h2>
+          <img class="divider" src="resources/images/background/gold-divider.png">
+          <p class="form-info first-info">If you’ve been to one of our restaurants, you’ve seen – and tasted – what keeps our customers coming back for more. Perfect materials and freshly baked food.
+          </p>
+          <br>
```
Comment (magicsd) [inline index.html]: Why?

File: index.html [hunk 2]
```diff
@@ -221,6 +221,53 @@ <h2 class="title review-title">Guest Reviews</h2>
...
+          <p class="form-info second-info">Delicious Lambda cakes,  muffins, and gourmet coffees make us hard to resist! Stop in today and check us out! Perfect materials and freshly baked food.
+          </p>
+          <form class="title form-field" action="#" method="POST">
+            <div class="field-container">
+              <div class="form-group">
+                <label class="input-title" for="name">Name</label> 
+                <br>
```
Comment (magicsd) [inline index.html]: Why?

File: index.html [hunk 3]
```diff
@@ -221,6 +221,53 @@ <h2 class="title review-title">Guest Reviews</h2>
...
+                <input class="input-field" id="name" name="name-field" type="text" placeholder="your name *" required>
+              </div>
+              <div class="form-group">
+                <label class="input-title" for="email">Email</label>
+                <br>
```
Comment (magicsd) [inline index.html]: Why?

File: index.html [hunk 4]
```diff
@@ -221,6 +221,53 @@ <h2 class="title review-title">Guest Reviews</h2>
...
+                <input class="input-field" id="email" name="email-field" type="text" placeholder="your email *" required>
+              </div>
+              <div class="form-group">
+                <label class="input-title" for="date">Date</label>
+                <br>
```
Comment (magicsd) [inline index.html]: Why?

File: index.html [hunk 5]
```diff
@@ -221,6 +221,53 @@ <h2 class="title review-title">Guest Reviews</h2>
...
+                <input class="input-field" id="date" name="date-field" type="text" placeholder="date *" required>
+              </div>
+              <div class="form-group">
+                <label class="input-title" for="party">Party number</label>
+                <br>
```
Comment (magicsd) [inline index.html]: Why?

File: resources/css/index.css [hunk 6]
```diff
@@ -385,3 +385,93 @@ div {
   font-size: 18px;
   margin-top: 30px;
 }
+
+.form-section {
+  background: url("../images/background/pattern-2.png");
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  background-image: url("../images/background/pattern-2.png");
```

File: resources/css/index.css [hunk 7]
```diff
@@ -385,3 +385,93 @@ div {
...
+.input-field::placeholder {
+  font-style: italic;
+}
+
+.input-field:focus {
+  outline: 1px solid #c89f2d;
+}
+
+.form-group:nth-child(1),
+.form-group:nth-child(2) {
```
Comment (magicsd) [inline resources/css/index.css]: Looks weird

File: resources/css/index.css [hunk 8]
```diff
@@ -385,3 +385,93 @@ div {
...
+  margin-bottom: 25px;
+}
+
+.submit-button {
+  background: #c89f2d;
```
Comment (magicsd) [inline resources/css/index.css]:
```suggestion
  background-colour: #c89f2d;
```

**Review by magicsd — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (katsu223) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (katsu223) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (katsu223) [reply index.html]: Fixed.

File: index.html (hunk 4 above)
Comment (katsu223) [reply index.html]: Fixed.

File: index.html (hunk 5 above)
Comment (katsu223) [reply index.html]: Fixed.

File: resources/css/index.css (hunk 6 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 7 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

File: resources/css/index.css (hunk 8 above)
Comment (katsu223) [reply resources/css/index.css]: Fixed.

**Review by magicsd — APPROVED** [review]

**Pull request merged**